grails-forge-core/src/main/java/org/grails/forge/feature/grailsProfiles/GrailsProfiles.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017-2024 original authors
```

Comment:
Technically 2024 makes the most sense.  Since it was copied and pasted from another feature and only some of it changed, it is possibly a gray area.

I updated to 2024.

Reply:
spotlessJavaCheck would not allow me to change them, so back to 2017-2024.

Execution failed for task ':grails-forge-core:spotlessJavaCheck'.
> The following files had format violations:
      src/main/java/org/grails/forge/feature/grailsProfiles/GrailsProfiles.java
          @@ -1,5 +1,5 @@
           /*
          -·*·Copyright·2024·original·authors
          +·*·Copyright·2017-2024·original·authors
           ·*
           ·*·Licensed·under·the·Apache·License,·Version·2.0·(the·"License");
           ·*·you·may·not·use·this·file·except·in·compliance·with·the·License.
      src/main/java/org/grails/forge/feature/grailsWrapper/GrailsWrapper.java
          @@ -1,5 +1,5 @@
           /*
          -·*·Copyright·2024·original·authors
74 actionable tasks: 39 executed, 26 from cache, 9 up-to-date
          +·*·Copyright·2017-2024·original·authors
           ·*
WARNING: The following functionality has been deprecated and will be removed in the next major release of the Develocity Gradle plugin. Run with '-Ddevelocity.deprecation.captureOrigin=true' to see where the deprecated functionality is being used. For assistance with migration, see https://gradle.com/help/gradle-plugin-develocity-migration.
           ·*·Licensed·under·the·Apache·License,·Version·2.0·(the·"License");
- The deprecated "gradleEnterprise.server" API has been replaced by "develocity.server"
           ·*·you·may·not·use·this·file·except·in·compliance·with·the·License.
- The deprecated "gradleEnterprise.buildScan.publishIfAuthenticated" API has been replaced by "develocity.buildScan.publishing.onlyIf { it.authenticated }"
  Run './gradlew :grails-forge-core:spotlessApply' to fix these violations.

Reply:
#392